tests/test_cli.py

```diff
@@ -263,6 +318,87 @@ def test_cli_start_with_input(runner):
                               'mcr.microsoft.com/azureiotedge-testing-utility:1.0.0-rc1'])
 
 
+@pytest.mark.skipif(get_docker_os_type() == 'windows', reason='The base image of edgeHubDev image is 1809 but agent is 1803.'
+                    'So it does not support windows container.')
+def test_cli_start_with_create_options_for_bind(runner):
+    test_resources_dir = os.path.join('tests', 'test_compose_resources')
+    try:
+        if get_docker_os_type() == "windows":
+            deployment_json_file_path = os.path.join(test_resources_dir, 'deployment_with_create_options_for_bind.json')
+            update_file_content(deployment_json_file_path, '/usr:/home/moduleuser/test',
+                                'C:\\\\\\\\\\\\\\\\Users:C:/moduleuser/test')
+        cli_setup(runner)
+        deployment_json_file_path = os.path.join(test_resources_dir, 'deployment_with_create_options_for_bind.json')
+        cli_start_with_deployment(runner, deployment_json_file_path)
+        wait_verify_docker_output(['docker', 'logs', 'edgeHubDev'], ['Opened link'])
+        wait_verify_docker_output(['docker', 'logs', 'tempSensor'], ['Sending message'])
+        if get_docker_os_type() == "windows":
+            output = start_process('docker exec -w C:/moduleuser/test tempSensor cmd && dir', True)
```

Comment:
Questions: why did you use `start_process` and `assert` for Windows while use `wait_verify_docker_output` for others?

Reply:
Updated code